core/options/manager.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Elementor\Core\Options;
+
+use Elementor\Core\Base\Module as BaseModule;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Manager extends BaseModule {
+
+	private $registered = [];
+
+	public function __construct() {
+		parent::__construct();
+
+		do_action( 'elementor/options/register' );
+	}
+
+	public function get_name() {
+		return 'options';
+	}
+
+	/**
+	 * @param Option_Base|string $classname
+	 */
+	public function register( $classname ) {
```

Comment:
Add an `unregister()` method so it'll be consistent with other managers

Reply:
In this case we won't the ability to unregister